README.md

```diff
@@ -2,29 +2,54 @@
 
 [![CircleCI Build Status](https://circleci.com/gh/all-of-us/workbench.svg)](https://circleci.com/gh/all-of-us/workflows/workbench)
 
-## Getting the code
+## Setup for Development
 
-* Run `git clone https://github.com/all-of-us/workbench`
+System requirements:
+
+  * [Docker CE](https://www.docker.com/community-edition)
+  * [Ruby](https://www.ruby-lang.org/en/downloads/)
+  * [Python](https://www.python.org/downloads/) >= 2.7.9
+  * [gcloud](https://cloud.google.com/sdk/docs/#install_the_latest_cloud_tools_version_cloudsdk_current_version)
+
+Docker must be installed to build and run code (For Google workstations, see
+go/installdocker.). Ruby is required to run our development scripts, which
+document common operations and provide a convenient way to perform them.
+Python is required by some scripts and the Google Cloud Tools.
+
+After you've installed `gcloud`, login using your `pmi-ops` account:
+
+```shell
+gcloud auth login
+```
+
+To initialize the project, run the following:
+
+```shell
+git clone https://github.com/all-of-us/workbench
+cd workbench
+git submodule init
```

Comment:
You can combine these as `git submodule update --init`.

Reply:
well TIL

Reply:
I was going to say that I'd prefer we automate this, but since it's so close in nature to cloning the repo, I'm fine with having it in the README. The PR makes the case :)